src/session_p.h

```diff
@@ -41,6 +41,17 @@
 /* number of all supported authentication methods */
 # define NC_SSH_AUTH_COUNT 3
 
+typedef enum {
+    LOCAL_DEFINITON = 0,
+    KEYSTORE_REFERENCE,
+    TRUSTSTORE_REFERENCE
+} KEYSTORE_TYPE;
```

Comment:
Rename to be libnetconf-specific. Values too.

Reply:
Renamed to NC_STORE_TYPE and values' names shortened